tests/foreman/maintain/test_service.py

```diff
@@ -23,9 +23,33 @@
 from robottelo.constants import HAMMER_CONFIG
 from robottelo.constants import MAINTAIN_HAMMER_YML
 from robottelo.constants import SATELLITE_ANSWER_FILE
+from robottelo.hosts import Satellite
 
 pytestmark = pytest.mark.destructive
 
+SATELLITE_SERVICES = [
+    "dynflow-sidekiq@.service",
+    "foreman-proxy.service",
+    "foreman.service",
+    "httpd.service",
+    "postgresql.service",
+    "pulpcore-api.service",
+    "pulpcore-content.service",
+    "pulpcore-worker@.service",
+    "redis.service",
+    "tomcat.service",
+]
+
+CAPSULE_SERVICES = [
+    "foreman-proxy.service",
+    "httpd.service",
+    "postgresql.service",
+    "pulpcore-api.service",
+    "pulpcore-content.service",
+    "pulpcore-worker@.service",
+    "redis.service",
+]
+
```

Comment:
Can we move this to constants please? I think some installer tests also uses same constants, so it can be kept at single place. If yes, then can you try if those tests can be updated as well?

Reply:
The list in installer is a little different that's why I decided to keep the constant in here. If we use this for another file then I'll move it to constants then. https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/installer/test_installer.py#L1296